app/src/main/java/com/sopt/clody/presentation/ui/home/screen/HomeScreen.kt

```
@@ -87,15 +90,21 @@ fun HomeRoute(
-            )
-        } else {
-            homeViewModel.updateSelectedDate(LocalDate.now())
+        try {
+            coroutineScope {
+                val calendarDeferred = async { homeViewModel.loadCalendarData(year, month) }
+                val dailyDeferred = async { homeViewModel.loadDailyDiariesData(year, month, day) }
+
+                calendarDeferred.await()
+                dailyDeferred.await()
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)